Flow_Over_Heated_Plate/Coupler_Solver.F90

```diff
@@ -23,21 +23,26 @@ SUBROUTINE Print(dataName,mesh,BCPerm,CoordVals)
         TYPE(Mesh_t), POINTER               :: mesh
         !------------------------Data Arrays----------------------------------------------
         REAL(KIND=dp), POINTER              :: CoordVals(:)
-        INTEGER, POINTER                    :: BCPerm(:)
+        INTEGER, POINTER                    :: BoundaryPerm(:)
```

Comment:
What was the thought behind this name change? Was it just to make it more clear as to what the variable is?